web/models/abilities.ex

```diff
@@ -90,6 +91,8 @@ defmodule Canary.Abilities do
 
     def can?(%User{} = user, :create, Skill), do: SkillPolicy.create?(user)
 
+    def can?(%User{} = user, :show, StripeAccount), do: StripeAccountPolicy.show?(user)
```

Comment:
We need a `can?` method here which will call `StripeAccountPolicy.show?(user, stripe_account)` instead of this one.

```
def can?(%User{} = user, :show, %StripeAccount{}=stripe_account), do: StripeAccountPolicy.show?(user, stripe_account)
```

This tells me that we ought to be testing the whole ability model. I had the same bug creep up on me in my own PR.